0_RootFS/Rootfs/bundled/conf/profile.d/bb_utils.sh

```diff
@@ -56,3 +56,15 @@ if [ -f /meta/.env ]; then
 	vecho_red "Loading previous environment..."
 	source /meta/.env
 fi
+
+# Quiet find
+qfind() {
+    find "$@" 2>/dev/null
+}
+
+# Function to install license files
+install_license () {
+    for file in "$@"; do
+	install -Dm644 "$file" "${prefix}/share/licenses/${SRC_NAME}/$(basename "${file}")"
+    done
+}
```

Comment:
We should make this a little more user-friendly; Let's add a usage message, like:

```
if [ $# -eq 0 ]; then
    echo "Usage: install_license license_file1.txt [license_file2.md, license_file3.rtf, ...]" >&2
    exit 1
fi
```

Reply:
Ugh, I tested various combinations (multiple arguments, files with spaces), but not the simplest one: no arguments :woman_facepalming:  Thanks for pointing it out!  I made the function also more verbose.

Reply:
Is your editor perhaps doing something strange with tabs here?  The whitespaces looks off in the github viewer.

Reply:
Yes, I used vim to do this edit :upside_down_face: